README.md

```diff
@@ -35,51 +37,18 @@ The ingress host name is follow, where `<base_hostname>` is your deployment's ba
 * Installation of the Ansible collections on the control node.
   * If installing from the Ansible Automation Hub, then run `ansible-galaxy install redhat.trusted_profile_analyzer`.
   * If installing from this Git repository, then clone it locally, and run `ansible-galaxy collection install -r requirements.yml`.
-* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
-* A PostgreSQL instance
-* SQS like [Kafka](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_streams/)
-* S3 service or S3 compatible service
-* Optional:
-  Installation of the `podman` binaries to verify that the RHTPA service is working as expected.
+    
+The following external services are provided by the customers.
```

Comment:
This sentence is also in passive voice.  Please rewrite in active voice, for example:

`You must provide the following external services:`

Also, since a list is proceeding the statement, end the sentence with a colon (`:`).